VideoLocker/res/layout/row_course_list.xml

```diff
@@ -126,48 +108,18 @@
                 tools:targetApi="17" />
         </LinearLayout>
 
-        <LinearLayout
-            android:id="@+id/schoolcode_layout"
+        <org.edx.mobile.view.custom.ETextView
+            android:id="@+id/course_detail"
```

Comment:
The ID here is `course_detail`, while the variable holding the inflated view is called `course_run`. We should probably use consistent naming here.